### PR TITLE
Add Hotjar heatmap - closes #196

### LINKED ISSUE #196: Add Hotjar for heatmap?
I got us a free lifetime account for [Hotjar](http://www.hotjar.com). Are you ok with me adding this to the site to see if it's helpful or not?  If not, we can always remove it later. Thoughts?

### PR DESCRIPTION
**Pull request by jdaudier**
ping @peterblazejewicz - might wanna check my work since it's really late - tks! And need your email to add you to Hotjar to see the stats.

### CODE REVIEW

File: src/sections/_footer.jade [hunk 1]
```diff
@@ -1,4 +1,6 @@
 footer
   p Made with &hearts; by&nbsp;
     a(href="https://github.com/marionettejs/marionettejs.com/graphs/contributors", target="_blank") all of us
-  a(href="#top", class="top") Top
+    a(href="//www.hotjar.com/?utm_source=badge", target="_blank")
+      img.hotjar-badge(src="//static.hotjar.com/b/hotjar-badge-light.png" alt="Hotjar - Unlimited insights from your web and mobile sites")
```
Comment (samccone) [inline src/sections/_footer.jade]: do we need this here?

Comment (samccone) [inline src/sections/_footer.jade]: like is it in their TOS

File: src/layouts/layout.jade [hunk 2]
```diff
@@ -30,6 +30,20 @@ html(lang='en')
...
     link(href='http://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css')
 
+    // Hotjar
+    script.
+      (function (f, b) {
+        var c;
+        f.hj = f.hj || function () {
+          (f.hj.q = f.hj.q || []).push(arguments)
+        };
+        f._hjSettings = {hjid: 13560, hjsv: 3};
```
Comment (samccone) [inline src/layouts/layout.jade]: @jdaudier can you share the login in the private chat plz

File: src/sections/_footer.jade (hunk 1 above)
Comment (jdaudier) [inline src/sections/_footer.jade]: We don't have to, but they shared it for giving us free lifetime access.  Want me to convert it to text only?

Comment (samccone) [inline src/sections/_footer.jade]: hmm good point, perhaps just some text on a new line would be fine.

Comment (jdaudier) [inline src/sections/_footer.jade]: you got it

File: src/layouts/layout.jade (hunk 2 above)
Comment (jdaudier) [inline src/layouts/layout.jade]: Sent you an invite to your gmail @samccone

Comment (samccone) [inline src/layouts/layout.jade]: awesome @peterblazejewicz of you can send either of us a ping with your email we can add you also

**Comment by samccone** [comment]
@jdaudier lets put this on the doc pages also

https://github.com/marionettejs/marionettejs.com/blob/master/src/docs/template.html
^^

**Comment by jdaudier** [comment]
ok converted to text on 1 line. I tried 2 lines but it looked worse.

**Comment by samccone** [comment]
sweet @jdaudier ming squashing this into 1 commit and I will get this in

File: src/layouts/layout.jade (hunk 2 above)
Comment (peterblazejewicz) [inline src/layouts/layout.jade]: @jdaudier @samccone sent via Gitter to Joanne

**Comment by peterblazejewicz** [comment]
Can we later consider this?

```diff
diff --git a/src/sections/_footer.jade b/src/sections/_footer.jade
index ea1698c..0f541c1 100644
--- a/src/sections/_footer.jade
+++ b/src/sections/_footer.jade
@@ -1,7 +1,7 @@
 footer
-  p Made with &hearts; by&nbsp;
+  p.lead Made with &hearts; by&nbsp;
     a(href="https://github.com/marionettejs/marionettejs.com/graphs/contributors", target="_blank") all of us
-    | .&nbsp; Insights by&nbsp;
-    a(href="//www.hotjar.com/?utm_source=badge", target="_blank") Hotjar
-    | .
-  a(href="#top", class="top") Top
\ No newline at end of file
+  p
+    small Insights by 
+      a(href="//www.hotjar.com/?utm_source=badge", target="_blank") Hotjar
+  a(href="#top", class="top") Top
```
It also better fits non-visual content (two distinct paragraphs). `p.lead` is optional, to help with additional styling if required.

**Comment by samccone** [comment]
yesss ^

**Comment by jdaudier** [comment]
@samccone it's 1 commit now.  After the merge, @peterblazejewicz, feel free to add your suggested change.

**Comment by samccone** [comment]
kk @peterblazejewicz if you want to add your PR on top of this

**Comment by jdaudier** [comment]
@peterblazejewicz Can you fix this while you're at it? I missed the word "with".

OLD: As of Feb 2015, this repo doesn't work Node v0.12.0.

NEW: As of Feb 2015, this repo doesn't work with Node v0.12.0.

**Comment by peterblazejewicz** [comment]
@samccone @jdaudier 
read/will do
Thanks!

**Pull request merged**